### ISSUE TITLE
IPv6 Example

### ISSUE THREAD

**Issue by davidkazuhiro** [CumulusNetworks/topology_converter#111]
Would there happen to be any examples of an IPv6 topology?
If not, I think it would be a good addition to the collection.

**Comment by davidkazuhiro** [comment]
No, I just meant in general. All the example dot files configure IPv4-only networks, but I think some IPv6-only or dual stack examples would be a good addition.

Of course, covering the oob scenario would be interesting too, I just wasn’t thinking of it specifically.

**Comment by ericpulvino** [comment]
Topology Converter (TC) tries to make a distinction between cabling and configuration. The primary goal of TC is to provide the cabling, the couple places where config is provided (such as the configuration provided as part of the `-c` workflow) is for the management of the devices and not really with the config of the devices themselves (ipv4 vs ipv6, bgp vs ospf etc).

Management planes continue to be IPv4 in many cases. I would certainly take a PR to provide an IPv6 management plane as part of the `-c` workflow.

But as far as TC is concerned cabling is cabling, whether you run IPv4 over it or IPv6 the cable looks the same.

Am I understanding your ask or did I get it wrong?

**Comment by davidkazuhiro** [comment]
Sorry I got a bit confused.

So what I think I'm asking for is an IPv6-only example, which would require `vagrant ssh-config` to return an IPv6 address for the VMs instead of IPv4 addresses.

For example, currently I get `127.0.0.1` from vagrant for ssh.

```$ vagrant ssh-config | grep HostName
  HostName 127.0.0.1
  HostName 127.0.0.1
```

This would need to be IPv6 if the VMs are configured with IPv4 switched off.

Does this relate to the Management Plane which can be configured with the `-c` workflow which you referred to?
